docs/datamodel/scalars/datetime.rst

```diff
@@ -182,6 +182,66 @@ EdgeDB stores and outputs timezone-aware values in UTC.
     :eql:type:`duration`.
 
 
+----------
+
+
+.. eql:type:: cal::relative_duration
+
+    A type representing a span of time.
+
+    Unlike :eql:type:`std::duration` a relative_duration is not an precise
+    measurment. For example:
+
+    .. code-block:: edgeql-repl
+
+        db> select (
+        ...     <datetime>'2020-01-01T00:00:00Z' +
+        ...     <cal::relative_duration>'1 year'
+        ... ) - <datetime>'2020-01-01T00:00:00Z';
+        {<duration>'8784:00:00'}
+        db> select (
+        ...     <datetime>'2019-01-01T00:00:00Z' +
+        ...     <cal::relative_duration>'1 year'
+        ... ) - <datetime>'2019-01-01T00:00:00Z';
+        {<duration>'8760:00:00'}
```

Comment:
Hm, this example is very hard to parse -- please add a few comments explaining what happened in the first query, what happened in the last, why the first result is that, and what's up with the second result.

Reply:
I think I improved this. Not sure it's optimal though.